### ISSUE TITLE
Thermal Printer TM-T82 Support

### ISSUE THREAD

**Issue by fiq265** [mike42/escpos-php#596]
I have printer model EPSON TM-T82 and not sure support with this package or not since you only listed for EPSON TM-T82II. If not support, I want to ask for help adding support for this printer. Thank you.

**Comment by mike42** [comment]
Yep, with the exception of some character encodings which vary between models (eg. Thai, Japanese and Indian scripts), the `default` profile works fine for all Epson TM-series printers.
